.env.geome

```diff
@@ -5,3 +5,4 @@ SOLR_VOLUME_NAME=solrdata_geome
 CRONTAB_FILE=crontab.geome
 UVICORN_ROOT_PATH=geome
 ISB_HOST=henry.cyverse.org
+ENV_FILE=geome.isb_web_config.env
```

Comment:
Have a new parameter to the docker build, which `.env` file to use when building the image.  Each `.env` flavor has a different metrics domain.